packages/nmap.vm/nmap.vm.nuspec

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>nmap.vm</id>
+    <version>7.93</version>
+    <authors>Fyodor, Nmap Project</authors>
+    <description>Port scanning utility and nc replacement with extended features</description>
+    <dependencies>
+      <dependency id="common.vm" />
+      <dependency id="npcap.vm" />
```

Comment:
I think I read nmap installs this automatically. Have you tested without this?

Reply:
It looks like nmap can install it without it being previous installed, but then we'd have to update the AutoHotkey installer script to handle both installers (npcap and nmap). I figured since we already have a standalone npcap package we could simply reuse it, this way the nmap installer doesn't need to install it as well.

Reply:
Ah, gotcha. Sounds good! Thank you!